code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -469,3 +469,83 @@ Contains:
 	armor = list("melee" = 40, "bullet" = 20, "laser" = 20,"energy" = 30, "bomb" = 60, "bio" = 100, "rad" = 90, "fire" = 85, "acid" = 75)
 	slowdown = 0.5
 	w_class = WEIGHT_CLASS_NORMAL
+
+//The funny power rangers.
+/obj/item/clothing/suit/space/hardsuit/ert/lp
+	name = "Loss Prevention Lieutenant Hardsuit"
```

Comment:
item names are usually lowercase, is there any specific reason for these not to be?

Reply:
Funny titles. I suppose its just habitual. No not really in particular

Reply:
These are mostly titles so capitalization isn't out of place I would say.